### PR TITLE
BankManagementApplication using SpringBoot

### PR DESCRIPTION
**Pull request by kumarkaustubh18**
Bank Management Application to get Customer details by passing Customer id in request Body

### CODE REVIEW

File: src/main/java/com/Example/BankManagement/Controller/CustomerController.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/customers")
+public class CustomerController {
+    @Autowired
+    private CustomerService customerService;
+
+    @PostMapping("/details")
+    public ResponseEntity<Optional<Customer>> getCustomerDetails(@RequestBody Customer id) {
```
Comment (manisha2310) [inline src/main/java/com/Example/BankManagement/Controller/CustomerController.java]: this should return Customer not Optional<Customer>.

**Review by manisha2310 — CHANGES_REQUESTED** [review]

File: src/main/java/com/Example/BankManagement/Entity/Account.java [hunk 2]
```diff
@@ -1,14 +1,13 @@
-package customerRegistration.coreJava.src.main.com.tutorial.projectwithMaven;
+package com.Example.BankManagement.Entity;
 
-import javax.persistence.Table;
 
-import javax.persistence.Entity;
-import javax.persistence.Id;
+import jakarta.persistence.Entity;
```
Comment (manisha2310) [inline src/main/java/com/Example/BankManagement/Entity/Account.java:4]: why use jakarta when already use javax class.

Comment (kumarkaustubh18) [reply src/main/java/com/Example/BankManagement/Entity/Account.java:4]: javax.persistence-api been replaced by jakarta.persistence-api in spring data jpa starter in year 2019 due to some trademark reason

File: src/main/java/com/Example/BankManagement/Controller/CustomerController.java (hunk 1 above)
Comment (kumarkaustubh18) [reply src/main/java/com/Example/BankManagement/Controller/CustomerController.java]: updated return type

**Review by manisha2310 — APPROVED** [review]

**Pull request merged**